apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -3,19 +3,22 @@
  * so this can only be used in cases where we have a redux store.
  */
 
-var React = require('react');
-var connect = require('react-redux').connect;
+import React, {PropTypes} from 'react';
+import {connect} from 'react-redux';
+import $ from 'jquery';
 
-var i18n = require('@cdo/locale');
-var commonStyles = require('../commonStyles');
-var styleConstants = require('../styleConstants');
-import {ConnectedWatchers} from './watchers/Watchers';
-var PaneHeader = require('./PaneHeader');
-var PaneSection = PaneHeader.PaneSection;
-var PaneButton = PaneHeader.PaneButton;
-var SpeedSlider = require('./SpeedSlider');
-import {setStepSpeed} from '../redux/runState';
-import ProtectedStatefulDiv from '../templates/ProtectedStatefulDiv';
+import i18n from '@cdo/locale';
+import Radium from 'radium';
+import commonStyles from '../../../commonStyles';
+import styleConstants from '../../../styleConstants';
+import {ConnectedWatchers} from '../../../templates/watchers/Watchers';
+import PaneHeader from '../../../templates/PaneHeader';
+const {PaneSection, PaneButton} = PaneHeader;
```

Comment:
This could also be `import PaneHeader, { PaneSection, PaneButton } from '../../../templates/PaneHeader'`